static/app/views/starfish/views/spanSummaryPage/sampleList/index.tsx

```diff
@@ -37,12 +39,23 @@ export function SampleList({groupId, transactionName, transactionMethod}: Props)
         groupId={groupId}
         transactionName={transactionName}
         transactionMethod={transactionMethod}
+        onClickSample={span => {
+          router.push(
+            `/performance/${span.project}:${span['transaction.id']}/#span-${span.span_id}`
+          );
+        }}
+        onMouseOverSample={sample => highlightSample(sample.span_id)}
+        onMouseLeaveSample={() => highlightSample(undefined)}
+        highlightSpanId={highlightedSpanId}
```

Comment:
Similar mismatch here, why not
```suggestion
        highlightedSpanId={highlightedSpanId}
```